core/src/ast/expr.rs

```diff
@@ -35,11 +35,13 @@ pub enum Expr {
         expr: Box<Expr>,
         negated: bool,
         pattern: Box<Expr>,
+        escape_char: Option<char>,
```

Comment:
I thought it would be better to follow mysql's way which sets default value of escape_char as `\` so that it does not need to be `Option`

Cuz `\` is the most commonly used.

> To test for literal instances of a wildcard character, precede it by the escape character. If you do not specify the ESCAPE character, \ is assumed

https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html#operator_like

Reply:
@devgony 
The changes pushed.